configs/example/gpufs/amd/AmdGPUOptions.py

```diff
@@ -247,16 +247,88 @@ def addAmdGPUOptions(parser):
         default="simple",
         help="register allocation policy (simple/dynamic)",
     )
+
+    parser.add_argument(
+        "--FunctionalTLB",
+        action="store_true",
+        help="Assumes TLB has no latency",
+    )
+
     parser.add_argument(
         "--register-file-cache-size",
         type=int,
         default=0,
         help="number of registers in cache",
     )
+
     parser.add_argument(
         "--memtime-latency",
         type=int,
         # Set to a default of 41 from micro-benchmarks
         default=41,
         help="Latency for memtimes in scalar memory pipeline.",
     )
+
+    parser.add_argument(
+        "--dgpu",
+        action="store_true",
+        default=False,
+        help="Configure the system as a dGPU instead of an APU."
+        "The dGPU config has its own local memory pool and is not "
+        "coherent with the host through hardware.  Data is "
+        "transfered from host to device memory using runtime calls "
+        "that copy data over a PCIe-like IO bus.",
+    )
```

Comment:
Is this not implied? Are you working on APU support for GPUFS?

Reply:
Removed this now. Same as above, I missed catching this when I reviewed the commit internally